src/services/TestDataService.ts

```diff
@@ -1255,13 +1362,14 @@ export class TestDataService {
                 }
             }
         });
-        const orderDelivery = this.getBasicOrderDeliveryStruct(deliveryState, shippingMethod, customerAddress)
-        let shippingCosts = 0;
-        if (orderDelivery.shippingCosts != null) {
-            shippingCosts = orderDelivery.shippingCosts.totalPrice;
-        }
+        const orderDelivery = this.getBasicOrderDeliveryStruct(deliveryState, shippingMethod, customerAddress);
+        orderDelivery.shippingCosts ? shippingCosts = orderDelivery.shippingCosts.totalPrice : shippingCosts = 0;
```

Comment:
@Phil23 is that possible to define somehow that `orderDelivery.shippingCosts` could not be undefined?

Reply:
This should be easier to read:

```suggestion
        const shippingCosts = orderDelivery.shippingCosts?.totalPrice ?? 0;
```